x-pack/plugins/endpoint/common/models/event.ts

```diff
@@ -22,10 +22,33 @@ export function eventTimestamp(
   }
 }
 
+/** TODO, seems wrong */
 export function eventName(event: EndpointEvent | LegacyEndpointEvent): string {
   if (isLegacyEvent(event)) {
     return event.endgame.process_name ? event.endgame.process_name : '';
   } else {
     return event.process.name;
   }
 }
+
+export function eventId(event: ResolverEvent) {
+  if (isLegacyEvent(event)) {
+    return String(event.endgame.serial_event_id);
+  }
+  return event.event.id;
+}
+
+export function entityId(event: ResolverEvent) {
+  if (isLegacyEvent(event)) {
+    return String(event.endgame.unique_pid);
+  }
+  return event.process.entity_id;
+}
+
+export function parentEntityId(event: ResolverEvent) {
+  if (isLegacyEvent(event)) {
+    const ppid = event.endgame.unique_ppid;
+    return String(ppid); // if unique_ppid is undefined return undefined
```

Comment:
❔ This will return the _string_ "undefined" if ppid is undefined, right? Maybe put quotes around `if unique_ppid is undefined return "undefined"`